Tilix4/OpenPype
# Bugfix: datablock override lib is None

## Changelog Description
Fixing `Nonetype object has no attribute reference` error in build by checking if `override_library` exists before accessing it.

## Testing notes
- Build e101 sq003 sh006.
- Build should be successful and should not raise any error.
